lms/djangoapps/ci_program/utils.py

```diff
@@ -62,6 +62,7 @@ def get_student_deadlines_from_zoho_data(zoho_record):
         student_data.values(),
         key=lambda k: (k.get('submission_deadline') or ''))
 
+    print(sorted_student_data)
```

Comment:
delete if not needed

Reply:
Don't know why that's still there. I actually saw that and thought I had taken it out. Anyways, removed now.